src/main/java/carracing/domain/car/Cars.java

```diff
@@ -0,0 +1,15 @@
+package carracing.domain.car;
+
+import java.util.List;
+
+public class Cars {
+    private List<Car> cars;
+
+    public Cars(List<Car> cars) {
+        this.cars = cars;
```

Comment:
방어적 복사에 대해 알게 되었으니 활용해보는건 어떠신가요 ? 
```suggestion
        this.cars = Collections.unmodifiableList(cars);
```

Reply:
https://johngrib.github.io/wiki/defensive-copy.md/